manage-data/data-store/data-streams/failure-store.md

```diff
@@ -0,0 +1,848 @@
+---
+applies_to:
+  stack: ga 8.19.0
+  serverless: ga 9.1.0
+---
+
+# Failure store [failure-store]
+
+A failure store is a secondary set of indices inside a data stream, dedicated to storing failed documents. A failed document is any document that, without the failure store enabled, would cause an ingest pipeline exception or that has a structure that conflicts with a data stream's mappings. In the absence of the failure store, a failed document would cause the indexing operation to fail, with an error message returned in the operation response.
+
+When a data stream's failure store is enabled, these failures are instead captured in a separate index and persisted to be analysed later. Clients receive a successful response with a flag indicating the failure was redirected. Failure stores do not capture failures caused by backpressure or document version conflicts. These failures are always returned as-is since they warrant specific action by the client.
+
+## Set up a data stream failure store [set-up-failure-store]
+
+Each data stream has its own failure store that can be enabled to accept failed documents. By default, this failure store is disabled and any ingestion problems are raised in the response to write operations.
+
+### Set up for new data streams [set-up-failure-store-new]
+
+You can specify in a data stream's [index template](../templates.md) if it should enable the failure store when it is first created.
+
+:::{note}
+Unlike the `settings` and `mappings` fields on an [index template](../templates.md) which are repeatedly applied to new data stream write indices on rollover, the `data_stream_options` section of a template is applied to a data stream only once when the data stream is first created. To configure existing data streams, use the put [data stream options API](indices-put-data-stream-options).
```

Comment:
Will the `indices-put-data-stream-options` link work? Or do we need to fully-qualify it with the URL?